WinUIGallery/ControlPages/AnimatedIconPage.xaml

```diff
@@ -21,17 +22,28 @@
                         For guidance on how to properly structure your animation file see the AnimatedIcon Guidance page.<LineBreak></LineBreak>
                     </TextBlock>
                     <Button PointerEntered="Button_PointerEntered" PointerExited="Button_PointerExited" Width="75" AutomationProperties.Name="AnimatedIcon Example">
-                        <AnimatedIcon x:Name="SearchAnimatedIcon">
-                            <AnimatedIcon.Source>
-                                <animatedvisuals:AnimatedFindVisualSource/>
-                            </AnimatedIcon.Source>
+                        <AnimatedIcon x:Name="SearchAnimatedIcon" Source="{x:Bind pages:AnimatedIconPage.GetAnimationSourceFromString(AnimatedVisualSourceSelection.SelectedItem), Mode=OneWay}">
                             <AnimatedIcon.FallbackIconSource>
                                 <SymbolIconSource Symbol="Find"/>
                             </AnimatedIcon.FallbackIconSource>
                         </AnimatedIcon>
                     </Button>
                 </StackPanel>
             </local:ControlExample.Example>
+            <local:ControlExample.Options>
+                <ComboBox x:Name="AnimatedVisualSourceSelection" Header="Kind" SelectedIndex="4" VerticalAlignment="Center">
```

Comment:
Can you add a MinWidth property to the ComboBox, so that the layout doesn't jump around with different text lengths?